easybuild/easyconfigs/g/genometools/genometools-1.5.8-foss-2015b-no-cairo.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'genometools'
+version = '1.5.8'
+versionsuffix = '-no-cairo'
```

Comment:
not sure if this is needed, someone else can dive in to include Cairo as a dependency, and it would only enhance the build?